source/common/stats/thread_local_store.cc

```diff
@@ -105,9 +101,8 @@ void ThreadLocalStoreImpl::mergeHistograms(PostMergeCb merge_complete_cb) {
     merge_in_progress_ = true;
     tls_->runOnAllThreads(
         [this]() -> void {
-          for (ScopeImpl* scope : scopes_) {
-            for (const auto& name_histogram_pair :
-                 tls_->getTyped<TlsCache>().scope_cache_[scope].histograms_) {
+          for (const auto& scopes : tls_->getTyped<TlsCache>().scope_cache_) {
```

Comment:
scopes or scope?  Previously it was 'scope' with a concrete type. WDYT of using a concrete type for this one as it's not super-obvious locally what type this resolves to, and I suspect it's not particularly verbose.

Reply:
Sure will do. Caveat being that my computer is Fd and I'm having trouble doing anything real so probably will need to wait till Monday.

Reply:
OK go ahead and merge then...I can do these myself easily after you merge.  I know we're trying to try these out so no need to delay a week for a  style nit.